docs/administrator-documentation/references/repos-csv.md

```diff
@@ -0,0 +1,119 @@
+| gradleArgs  | `false`  | Build arguments that are added to the end of the Gradle command line when building LSTs.                                                                                                                                                               | `-Dmyprop=myvalue`                                                            |
+| org*        | `false`  | If you want to configure an organizational hierarchy, you can provide one or more organization columns. Each column will specify an organization the repository should be part of. The column name should be `org` plus a number such as: `org1,org2,org3`.  | `openrewrite`                                                           |
+</details>
+
+## Generating this file
+
+To kick-start a `repos.csv` file, we recommend using "[repo fetchers](https://github.com/moderneinc/repository-fetchers)". Repo fetchers are scripts that generate a CSV file of all of your repositories by making API calls to your source code manager.
+
+## Defining an organizational hierarchy
+
+Org hierarchy is best explained with an example. Consider an organization that consists of two teams and two directors:
+
+```
+All
+├── Director A
+|       └── Team 1
+|       └── Team 2
+├── Director B
+
+```
+
+The following CSV file would represent this organizational structure:
+
+```bash showLineNumbers
+cloneUrl,branch,org1,org2,org3
+"https://github.com/apache/maven-doxia","master","Team 1","Director A","ALL"
+"https://github.com/Netflix/photon","main","Team 2","Director A","ALL"
+"https://github.com/Netflix/ribbon","master","Director A","ALL"
+"https://github.com/apache/maven-doxia","master","Director B","ALL"
+```
+
+In the above file, we define 5 organizations (ALL, Director A, Director B, Team 1, and Team 2) and 3 repositories. Notice that one repository can be defined multiple times if you want two distinct organizations to have access to it.
+
+Also note that organizations on the left are children of organizations on the right. For instance, `Team 1` is a child of `Director A` which is a child of `ALL`.
+
+:::tip
+Comma-separated files (CSV) are best manipulated using a dedicated editor such as Microsoft Excel, Google Sheets, or Apple Numbers. Here is what the above CSV would look like on Apple Numbers:
+
+![](./assets/numbers-csv-screenshot.png)
+:::
+
+## Formatting details
+
+- A single row may not have more columns than what is defined in the first row.
+- A single repository may be associated with multiple organizations by being referenced in multiple rows. In the above
+  example, `maven-doxia` is associated with both `Director B` and `Team 1`.
+- Repositories can be associated with any organization down the tree, not just the leaves. See line 4 in the `csv` file
+  for an example.
+
+:::warning
+Org columns must start immediately after the branch column. Likewise, they must be contiguous. **Do not** add empty orgs
+so that the orgs line up with one another.
```

Comment:
Most of the warnings you have here are no longer true. You can put the orgs at the end or in the middle. You can also put empty orgs at the beginning or end so that all the orgs line up. The only thing you can't do is have empty orgs in the middle, and even _that_ may work correctly (ignoring the empty ones), I just haven't tested it.

```suggestion
**Do not** add empty orgs in between organizations so that the orgs line up with one another.
```